CodeEdit/NavigatorSidebar/ProjectNavigator/OutlineView/OutlineViewController.swift

```diff
@@ -207,9 +208,9 @@ extension OutlineViewController: NSOutlineViewDelegate {
     /// Recursively gets and selects an ``Item`` from an array of ``Item`` and their `children` based on the `id`.
     /// - Parameters:
     ///   - id: the id of the item item
-    ///   - collection: the array to search for
-    private func select(by id: Item.ID, from collection: [Item]) {
-        guard let item = collection.first(where: { $0.id == id }) else {
+    ///   - collection: the array to searchx for
```

Comment:
A little typo in the doc.

Reply:
Fixed in [329757c](https://github.com/CodeEditApp/CodeEdit/pull/580/commits/329757c5a1231022ab9697a9686cd78e60f896f5)